packages/typespec-vscode/src/const.ts

```diff
@@ -0,0 +1,9 @@
+import { SettingName } from "./types.js";
+
+export const EmitterSettingName: Record<string, string> = {
```

Comment:
this is not a const. Please move it with the feature that using it. Or create a settings.ts file and more all the settings definition there. thx

Reply:
This is a record to hold the map between the language type with the settingName. It is a data, not type.

Reply:
removed. not used.